src/main/java/org/hydev/mcpm/client/uninstallUseCase/Uninstaller.java

```diff
@@ -0,0 +1,36 @@
+package org.hydev.mcpm.client.uninstallUseCase;
+import java.io.File;
+import java.util.Arrays;
+import org.hydev.mcpm.client.injector.PluginLoader;
+import org.hydev.mcpm.client.injector.PluginNotFoundException;
+
+
+public class Uninstaller implements UninstallBoundary{
+    private boolean remove_depen;
+
+
+    public void uninstall(String name) throws PluginNotFoundException {
+        PluginLoader pil = new PluginLoader();
```

Comment:
This is concrete, you should take an `UnloadBoundary` as part of an `UninstallInput` object instead of creating a plugin loader like this.

Generally we want the uninstall method to work with *any* way to unload plugins, so if we want to test it or if we decide to switch how we want to unload stuff it becomes super easy.